test/share-socket.js

```diff
@@ -26,6 +26,7 @@ describe('share-socket', function() {
   })
 
   afterEach(function(done) {
+    this.skip()
     this.timeout(200)
     setTimeout(function() {
       server.close(done)
```

Comment:
I actually narrowed down the problem now to this `afterEach` hook. If it is skipped the tests pass on Mac OS. Maybe increasing the timeouts could fix the problem?

Reply:
Hmm. I did some more investigation and somehow the tests in `share-socket.js` fail randomly with the following error message:

` Caught exception: Error [ERR_SOCKET_DGRAM_NOT_RUNNING]: Not running`

Maybe this is a more general problem with UDP sockets in NodeJS under Mac OS?